bundles/specmate-integration-test/src/com/specmate/test/integration/ModelGenerationTestEn.java

```diff
@@ -197,21 +197,21 @@ public void testModelGenerationEN12_or_1() {
 	/*
 	 * Problems: - removes "-" in "decision-node"
 	 */
-	@Test
-	public void testModelGenerationEN13_or_2() {
-		String text = "If the model contains an edge, or a node, or a decision-node, Specmate displays the details at the right.";
-		RequirementsFactory f = RequirementsFactory.eINSTANCE;
-		CEGModel model = f.createCEGModel();
-		CEGNode node1 = createNode(model, "the model", "contains an edge", NodeType.AND);
-		CEGNode node2 = createNode(model, "the model", "contains a node", NodeType.AND);
-		CEGNode node3 = createNode(model, "the model", "contains a decision node", NodeType.AND);
-		CEGNode node4 = createNode(model, "Specmate", "displays the details at the right", NodeType.OR);
-		createConnection(model, node1, node4, false);
-		createConnection(model, node2, node4, false);
-		createConnection(model, node3, node4, false);
-		JSONArray generated = generateCEGWithModelRequirementsText(text);
-		checkResultingModel(generated, model);
-	}
+//	@Test

```

Comment:
Instead of commenting out the test, we should add the `@Ignore`-annotation.

Reply:
done - added ignore annotations